es-design-system/pages/examples/form-validation.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+    <b-container>
+        <b-row>
+            <b-col>
+                <h1>
+                    Form with (form level) Validation
+                </h1>
+                <p>
+                    See
+                    <b-link
+                        :to="{ name: 'examples-form-field-validation___en' }">
+                        Form Field Validation
+                    </b-link> for field level validation example
+                </p>
+                <h2>
+                    UX Guidelines
+                </h2>
+                <p>
+                    <ul>
+                        <li>
+                            Show loading spinner on button during server request
+                        </li>
+                        <li>
+                            Use <b-link :to="{ name: 'molecules-es-form-msg___en' }">
+                                EsFormMsg
+                            </b-link>
+                            for displaying of server-side form-level errors
+                        </li>
+                    </ul>
+                </p>
+            </b-col>
+        </b-row>
+        <b-row class="border-top pt-4 my-2">
+            <b-col
+                cols="12"
+                lg="8">
+                <b-form
+                    @submit.stop.prevent="onSubmit">
+                    <es-form-msg
+                        class="mt-3"
+                        :variant="formMsgVariant"
+                        :message="formMsg"
+                        @hidden="formMsg = ''" />
+                    <es-form-input
+                        id="form-input-name"
+                        v-model="form.name"
+                        :disabled="isSubmitInProgress"
+                        required>
+                        <template #label>
+                            Name
+                        </template>
+                    </es-form-input>
+                    <div class="d-flex flex-grow-1 justify-content-end mt-3">
+                        <es-button
+                            type="submit"
+                            class="w-100 w-lg-auto"
+                            :loading="isSubmitInProgress"
```

Comment:
This `loading` prop currently does nothing, yes? I can't find it in EsButton or in the Bootstrap Vue button docs.

Is it a hint towards how we want to implement this spinner in the future, as part of EsButton? (which I'd agree with)

Reply:
No, I was just copying what was in `pages/organisms/es-form.vue`. Fixed via b121f55c63649a80f17d7971d253fcbbfc88e5b5